ext/timers/lib.rs

```diff
@@ -74,12 +80,38 @@ where
   Ok(result)
 }
 
+pub struct TimerHandle(Rc<CancelHandle>);
+
+impl Resource for TimerHandle {
+  fn name(&self) -> Cow<str> {
+    "timerCancelHandle".into()
+  }
+
```

Comment:
This is a bit strange... why name it `timerCancelHandle` instead of just `timer`?

Reply:
I renamed the struct and then forgot to change the `name` method.

Should it be just `timer`, not `timerHandle`?

Reply:
I think `timer` is sufficiently self-explanatory.